src/Pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { Loader } from '../../components/Loader';
+
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+import { useParams } from 'react-router-dom';
+
+type Params = {
+  personSlug?: string;
+};
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [errorMessage, setErrorMessage] = useState(false);
```

Comment:
Consider changing the type of `errorMessage` from `boolean` to `string | null`. This allows you to store and display more detailed error messages instead of just a boolean flag.